### ISSUE TITLE
[Bug]: Crash on StartUp (MineTogether)

### ISSUE THREAD

**Issue by FTWCougar** [FTBTeam/FTB-Modpack-Issues#1621]
### Modpack

FTB StoneBlock 3

### Modpack version

1.0.2

### Log Files

---- Minecraft Crash Report ----
// There are four lights!

Time: 11/3/22, 9:04 PM
Description: Rendering overlay

```
java.util.ConcurrentModificationException: null
	at java.util.HashMap$HashIterator.nextNode(Unknown Source) ~[?:?] {}
	at java.util.HashMap$EntryIterator.next(Unknown Source) ~[?:?] {}
	at java.util.HashMap$EntryIterator.next(Unknown Source) ~[?:?] {}
	at net.minecraft.client.renderer.texture.TextureManager.m_118482_(TextureManager.java:190) ~[client-1.18.2-20220404.173914-srg.jar%23263!/:?] 
```
### Describe the bug

Launch game crashes at end of loading.

### Steps to reproduce

1. launch game
2. wait for load
3. game crashes

### Expected behaviour

load game with no crashes

**Comment by Raidobw2** [comment]
Remove MineTogether from your mods for now, the devs have identified the error and will update the modpack with a patch quickly. With the mod remove until you update again, you should be ok

**Comment by Sunekaer** [comment]
Should be fixed in 1.1.0
